tests/renderer/components/settings-electron-spec.tsx

```diff
@@ -223,4 +226,43 @@ describe('ElectronSettings component', () => {
       ]);
     });
   });
+
+  describe('disableDownload()', () => {
+    it('disables download button when return value is true', () => {
+      (disableDownload as any).mockReturnValueOnce(true);
+
+      const version = '3.0.0';
+      const ver = {
+        source: VersionSource.remote,
+        state: VersionState.unknown,
+        version,
+      };
+
+      store.versions = { version: ver };
+      store.versionsToShow = [ver];
+
+      const wrapper = shallow(<ElectronSettings appState={store as any} />);
+
+      expect(wrapper.find('.disabled-version')).toHaveLength(1);
+    });
+
+    it('disables download button when return value is true', () => {
```

Comment:
These two tests have the same name and are very similar. Does there need to be two here?

Reply:
The tests were simply to make sure disableDownload() works for cases where there is a single download button and cases where there are multiple buttons. Nothing else in particular. I have removed one of the tests because it does look like they are trying to achieve the same thing.